packages/@jsii/utils/lib/assembly.ts

```diff
@@ -0,0 +1,98 @@
+import * as spec from '@jsii/spec';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import * as zlib from 'zlib';
+
+export const SPEC_FILE_NAME = '.jsii';
```

Comment:
I could be wrong, but my intuition is that this constant belongs in this module, not `@jsii/spec`. Removing it from `@jsii/spec` is a breaking change, but I figured we could just start using this constant, and duplication of one line never hurt anyone.

Reply:
You can actually move the constant there AND re-export it from `@jsii/spec` (or vice-versa if the dependency can flow in that direction), too (for convenience). The file name is basically considered part of the contract between the various libraries/CLIs in the jsii toolchain, and so in this sense it is part of the spec.